tests/ttnn/nightly/unit_tests/operations/conv/test_conv2d.py

```diff
@@ -292,6 +292,8 @@ def run_conv(
             return_output_dim=True,
             return_weights_and_bias=True,
         )
+    assert tt_output_tensor_on_device.layout == output_layout
```

Comment:
this doesn't have to be true if user has provided memory config which overrides the layout?